thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/ThirdEyeDetectorApplication.java

```diff
@@ -56,12 +58,17 @@
   private static final Logger LOG = LoggerFactory.getLogger(ThirdEyeDetectorApplication.class);
 
   public static void main(final String[] args) throws Exception {
-    String thirdEyeConfigDir = args[0];
```

Comment:
This change is actually reverse compatible, but was made so that users can run a wider variety of commands other than `server` (eg `db migrate`). Now the only restriction on the commands that can be run via this main method is that the last argument must be the config directory - I'll add a comment indicating this.